crates/bindings-csharp/Codegen.Tests/fixtures/diag/Lib.cs

```diff
@@ -1,5 +1,7 @@
 using SpacetimeDB;
 
+#pragma warning disable STDB_UNSTABLE // Enable experimental SpacetimeDB features
```

Comment:
Here in `diag` you should actually be able to leave this enabled. You should the relevant error in the diagnostics snapshots. (Unless this particular error subsystem is different from all the other ones, but the Roslyn stuff seems pretty well designed so I don't think it will be.)